grakn-core/src/main/java/ai/grakn/concept/Type.java

```diff
@@ -38,7 +38,7 @@
  * @see Role
  * @see RelationshipType
  * @see AttributeType
- * @see RuleType
+ * @see Rule
```

Comment:
This isn't a `Type` so shouldn't be here!

Reply:
🍌